2418_Sort_the_People.py

```diff
@@ -0,0 +1,13 @@
+# O(n) T(n)
+from typing import List
+
+
+class Solution:
+    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
+        result = []
+        
+        heights_sorted = sorted(heights, reverse=True)
+        for height in heights_sorted:
+            index = heights.index(height)
```

Comment:
The index method of list is O(n), so it's inefficient. Change this part so that this program is more efficient.

Reply:
@Ryuju0911 
I have a question. I searched the solutions. But, I couldn't find the answer.
In my opinion, I have to match both index and heights and name. So, I think I would use For statement for that. I couldn't' t find the way to make index method more effective.
Please tell me the way or somethings